gammapy/catalog/hawc.py

```diff
@@ -196,11 +170,118 @@ class SourceCatalog2HWC(SourceCatalog):
     description = "2HWC catalog from the HAWC observatory"
     """Catalog description"""
 
-    source_object_class = SourceCatalogObject2HWC
+    source_object_class = SourceCatalogObjectHWCBase
```

Comment:
No, here the object class is still `SourceCatalogObject2HWC`, not the Base class